src/tactic/expand_exists.lean

```diff
@@ -195,22 +219,52 @@ Note that without the last argument `nat_greater_nonzero`, `nat_greater_lt` woul
 ```lean
 #check nat_greater_lt -- nat_greater_lt : ∀ (n : ℕ), n < nat_greater n ∧ nat_greater n ≠ 0
 ```
+
+All definitions will be created in the same namespace as the `exists` lemma. You can prepend the
+name with `@` to create it in the root namespace:
```

Comment:
Instead of looking for an optional `@`, maybe it would be better to just parse a `name` and check if the first component of the name is `_root_`. Lean already supports the syntax `def _root_.foo` for escaping the current namespace, so this will be familiar.

Reply:
I agree, that makes more sense.